colaraz/cms/static/js/header.js

```diff
@@ -31,6 +31,20 @@ function headerToggleDropdown() {
     });
 }
 
+function openInviteFriendPopup(inviteFriendUrl) {
+    let width = 802;
+    let height = 554;
+    let screenWidth = this.window.screen.width;
+    let screenHeight = this.window.screen.height;
+    let windowSize = `width=${width}, height=${height}`;
+    if (screenWidth < width || screenHeight < height){
+        windowSize = `width=${screenWidth}, height=${screenHeight}`;
+    }
+    let windowTitle = 'Colaraz - Invite Friends';
+    let newWindow = window.open(inviteFriendUrl, 'Invite Friend', windowSize);
+    setTimeout(() => newWindow.document.title = windowTitle, 0);
```

Comment:
I think this will work without the setTimeout too. But please verify and confirm this as I could be wrong.

Reply:
It gives error sometimes that's why I used it.